k8s/nginx/deployment.yml

```
@@ -0,0 +1,19 @@
+    matchLabels:
+      app: nginx-deployment
+  template:
+    metadata:
+      labels:
+        app: nginx-deployment
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:latest
+          ports:
+            - containerPort: 80
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)